src/components/GameBoard.vue

```diff
@@ -15,12 +15,27 @@ export default {
       disableDeck: false,
       targetWord: "",
       startTime: "",
+
     };
   },
   mounted() {
     this.shuffleCards();
   },
   methods: {
+    startGame() {
+      console.log("Game started!");
+
+      this.targetWord = "Match The Cards";
+      this.shortmessage.innerHTML = `<span>${targetWord}</span>`;
```

Comment:
`this.shortmessage` is a reactive state/variable of the gameboard component. It's not a reference to an element, so setting `this.shortmessage.innerHTML` won't behave as expected. 

Instead, you want to change this to `this.shortmessage = 'Match The Cards'`. Then, in the [template](https://github.com/BerylOdoyo/memory-card-game-vue/pull/2/commits/fc2edd99ac2b078edcd4d508d202cb8c52c5ffb0#diff-992d2cbf1fff1604e234506cac561197fef6c1d37cea82184cb84f9fb3a91826R115) you can use data binding in the template to show the message. Whenever you update `this.shortmessage` to something else (e.g. in `this.gameOver()`), the message in the template will change!
 
e.g.

```
<template>
  <p>{{ shortmessage }}</p>
  <div class="game-board">
  ...
```

You'll also want to make sure you add it to the object returned in `data()`: https://github.com/BerylOdoyo/memory-card-game-vue/pull/2/commits/fc2edd99ac2b078edcd4d508d202cb8c52c5ffb0#diff-992d2cbf1fff1604e234506cac561197fef6c1d37cea82184cb84f9fb3a91826R10

See the below for different ways to do data binding:
- Text: https://vuejs.org/guide/essentials/template-syntax.html#text-interpolation
- Raw HTML: https://vuejs.org/guide/essentials/template-syntax.html#raw-html